src/njoy21/input/COVR/Card4/Mt1.hpp

```diff
@@ -0,0 +1,15 @@
+struct Mt1 {
```

Comment:
Where is this tested?

Reply:
[Here](https://github.com/njoy/NJOY21/blob/feature/COVR/src/njoy21/input/COVR/Card4/test/Card4.test.cpp).